pom.xml

```diff
@@ -39,6 +39,9 @@
 		<!-- Note: Change ./spring-cloud-dataflow-server-local/pom.xml to match the spring-cloud.version -->
 		<spring-cloud.version>Dalston.SR1</spring-cloud.version>
 
+		<!-- Used in docs. Change to the recommended release version (git branch/tag) -->
+		<spring-cloud-stream-apps.branch>master</spring-cloud-stream-apps.branch>
```

Comment:
Given SCDF doesn't depend on app starters, maybe it is better to stick to a specific `release` version instead of using `snapshot` (master branch) and update during the release?